src/zope/app/apidoc/utilities.rst

```diff
@@ -166,13 +166,15 @@ For interfaces we simply get
   >>> utilities.getPythonPath(ISample)
   'zope.app.apidoc.doctest.ISample'
 
-and for classes
+and for classes (note that we must pass a "bound method", because the
+unbound method is no longer present in Python 2):
 
   >>> class Sample(object):
   ...     def sample(self):
   ...         pass
 
-  >>> utilities.getPythonPath(Sample.sample)
+  >>> PY3 = str is not bytes
+  >>> utilities.getPythonPath(Sample().sample if PY3 else Sample.sample)
   'zope.app.apidoc.doctest.Sample'
```

Comment:
Python 3 gives you `__qualname__`, which should return '<Class>.<methodname>' for methods.

I imagine `getattr(possiblymethod, '__qualname__', possiblymethod.__name__)` should work on both Pythons (as long as you also keep the Python-2-only unbound-method handling bits, since Python 2 won't have `__qualname__`).